docs/start/api.tx.subs.md

```diff
@@ -0,0 +1,61 @@
+# Transactions and subscriptions
+
+Previously we send simple transactions with the `api.tx` endpoints, in this section we will extend that to monitor the actual transactions for inclusion and also extend the monitoring for transaction events.
+
+## Transaction finalization
```

Comment:
The use of "finalization" here just means included in a block right? This might be confusing for dApp developers who are also thinking of finalization in the sense of grandpa or casper FFG where it means the block will never be reverted or orphaned by a re-org.

Reply:
Kept the working to align with the actual enum, which is `Finalized`

Reply:
Clarified via comment.